flink-libraries/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/LocalExecutor.java

```diff
@@ -381,6 +433,23 @@ private Table createTable(TableEnvironment tableEnv, String query) {
 		}
 	}
 
+	/**
+	 * Applies the given update statement to the given table environment with query configuration.
+	 *
+	 * @param tableEnv table environment
+	 * @param queryConfig query configuration
+	 * @param statement SQL update statement (e.g. INSERT INTO)
+	 */
+	private void applyUpdate(TableEnvironment tableEnv, QueryConfig queryConfig, String statement) {
```

Comment:
rename `statement ` -> `updateStatement` and drop `@param` section from java doc?